source/openapi3/split.yaml

```diff
@@ -5605,5 +5629,54 @@ components:
             suburb: Lead
             state: NSW
             postcode: '2478'
+    SimulateIncomingPayIDPaymentRequest:
+      required:
+        - payid_email
+        - amount
+      type: object
+      properties:
+        payid_email:
+          type: string
+          max: 256
+          description: Receivable contact PayID email (256 max. characters)
```

Comment:
Any particular reason you're opting for `(256 max. characters)` instead of the standard `(Min: 6 - Max: 256)`
Also, some descriptions have a `.` at the end and others don't. Pick one :)

Reply:
Was just copying your examples on the max :D. Will update and also remove any extra `.`